godtools/App/Features/ToolSettings/Presentation/ToolSettings/Subviews/ToolSettingsOptions/ToolSettingsOptionsView.swift

```diff
@@ -28,15 +28,18 @@ struct ToolSettingsOptionsView: View {
             ScrollView(.horizontal, showsIndicators: false) {
                 HStack {
```

Comment:
I might make a separate ticket to add a ForEach here and do that later.  I should have originally done that.  We could then just check for toolOptions.isEmpty in ToolSettingsView which would be a little safer.